### DISCUSSION TITLE
Slightly modify sankey diagram

### DISCUSSION THREAD

**Discussion by UMHB-InfoSec** [Security-Onion-Solutions/securityonion#12106] [2.4]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

544

### RAM

2.1 TiB

### Storage for /

shared with nsm

### Storage for /nsm

200 TiB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Nice and simple question:

I love me some sankey diagrams, but I have a smidgen of an issue.  How can I change the spacing for the elements the diagram produces?  Specifically, if the dataset the diagram is attempting to draw gets even moderately medium the elements begin to stack on top of each other and become difficult to read:

![Screenshot from 2024-01-02 16-52-28](https://github.com/Security-Onion-Solutions/securityonion/assets/80428885/9d2da839-37d0-4913-b6a1-2e04bbc4fd48)

I would like to spread the items out, even if it means I have to scroll to see the rest of the graph.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by reyesj2** [comment]
Try the expand icon at the top of the sankey. It will give it more space to display text.

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/94730068/287979dd-2b56-48bc-ad7b-4e2c22478965)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/94730068/73999666-982e-4bf8-9bf6-ce0fb1226f3e)

**Reply by UMHB-InfoSec** [reply]
Ha!  I guess I should have mentioned that I had already done that.  My apologies.

**Reply by reyesj2** [reply]
Try creating a sankey with a more specific search. Even if you have to use multiple diagrams at least that way it'll be easier to view.
